src/features/controls.ts

```diff
@@ -202,8 +247,8 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
       this[$controls].removeEventListener('change', this[$changeHandler]);
     }
 
-    updated(changedProperties: Map<string, any>) {
-      super.updated(changedProperties);
+    updated(changedProperties: Map<string|number|symbol, unknown>) {
+      super.updated(changedProperties as Map<string, any>);
```

Comment:
Where did number, symbol and unknown come from? What's the difference between unknown and any?

Reply:
From [this Stack Overflow answer](https://stackoverflow.com/questions/51439843/unknown-vs-any)

> unknown which is the type-safe counterpart of any. Anything is assignable to unknown, but unknown isn't assignable to anything but itself and any without a type assertion or a control flow based narrowing. Likewise, no operations are permitted on an unknown without first asserting or narrowing to a more specific type.

In other words, `unknown` is an actual type (technically the type that includes all possible types) whereas `any` is just a dark void in which all type information is discarded.

This is probably preferable to `any` in many cases. I guess we should use it where we can.

The reason why we changed the type signature here was to satisfy a TypeScript compiler check. The actual signature for this method is the new one, and our decorator operates on any derivative of `UpdatingElement`, so it was causing problems that our signatures were not  lining up.